components/Navigation/navigation.css

```diff
@@ -0,0 +1,89 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+.box{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 60px;
+}
+.header{
+    margin-top: 20px;
+    display: flex;
+    width: 735px;
+    height: 61px;
```

Comment:
Any reason behind using px instead of relative units ?

Reply:
done